dashboard/src/actions/overviewActions.js

```diff
@@ -22,7 +22,7 @@ export const getDatasets = () => async (dispatch, getState) => {
     params.append("metadata", CONSTANTS.DASHBOARD_SAVED);
     params.append("metadata", CONSTANTS.DASHBOARD_SEEN);
     params.append("metadata", CONSTANTS.USER_FAVORITE);
-
+    params.append("metadata", "dataset.uploaded");
```

Comment:
It looks odd that this one doesn't have a constant like the others! Is it going to feel sad and left out? 😆

Reply:
I noted that, too, but I couldn't find anywhere else in the code that we referenced `"dataset.uploaded"`, so I figured we could get by without a symbolic reference for it.

Reply:
And, we need to re-do how define and reference constants (I'm thinking I might take that on...), so I let it pass.